frontend/server.ts

```diff
@@ -6,7 +6,6 @@ import * as express from 'express';
 import * as fs from 'fs';
 import * as path from 'path';
 import * as domino from 'domino';
-import { createProxyMiddleware } from 'http-proxy-middleware';
```

Comment:
Why was this removed?

Reply:
This import is not used anywhere in the file so I removed it.
Is there some kind of magic injection thingy with this import or can I get rid of it?

Reply:
I confirm this import is not used. Ref on how it is used: https://github.com/chimurai/http-proxy-middleware#tldr